fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -1461,7 +1461,7 @@ private CommandProcessingResult updateDatatableEntry(final String dataTableName,
             }
             if (columnName.equals("Cupo aprobado") || columnName.equals("Cupo")) {
                 if (columnValue instanceof Number) {
-                    cupo = BigDecimal.valueOf((Integer) columnValue);
+                    cupo = BigDecimal.valueOf(((Number) columnValue).intValue());
```

Comment:
why not new BigDecimal(columnValue) ?

Reply:
Sometimes the data type is different, in my local its integer. Cupo is not always big decimal, it can be integer so this way it will handle suppose that it changes.